src/components/ui-shell/_navigation-menu.scss

```diff
@@ -9,17 +9,22 @@
   // Navigation
   //----------------------------------------------------------------------------
   .#{$prefix}--navigation {
-    position: absolute;
+    position: fixed;
     top: mini-units(6);
     bottom: 0;
     left: 0;
     background-color: $ibm-colors__gray--90;
     width: mini-units(32);
-    z-index: 1000;
+    z-index: 7000;
```

Comment:
Should we use the `z()` helper with the floating panel z-index?

Reply:
I wasn't sure if we needed to use the helper based on the existing code and from searching for precedence throughout the codebase, but it makes sense to use it here